index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="css/styles.css">
```

Comment:
- [ ] The styles are not loading on the web page. Check how you are loading your stylesheet on this line.
![Screen Shot 2020-07-16 at 10 30 00 PM](https://user-images.githubusercontent.com/18187039/87729042-f1525680-c7b3-11ea-825e-dc645395078c.png)